src/config.yaml

```diff
@@ -4,12 +4,14 @@ canopen_vcan:
     bitrate: 500000
     nodes: [{local: false, eds_location: 'eds_files/Arduino1.eds', node_purpose: {name: 'Front view object distance'
     , type: 0}}]
+    max_speed: 3
```

Comment:
wat doet max_speed?

Reply:
max_speed geeft de maximale snelheid aan voor alle wheels. Eerder hadden MAX_SPEED als globale variabel en dat heb ik nu zegmaar in de config file geplaatst omdat het fijn is om 1 locatie te hebben om alle configuraties te kunnen aanpassen.

Reply:
op dit moment wordt het nog niet echt gebruikt omdat we nog geen actuatoren aansturen

Reply:
moeten we daarin de meeteenheid in opnemen? bijvoorbeeld `max_speed_in_ms` (meters per seconde)

Reply:
ja dat is goed ik zal het aanpassen